### PR TITLE
javascript-operators-and-expressions - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-01-30 at 12 09 33 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+console.log('value of is acidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+var headCount = 420;
+var isSparta = headCount === 300;
+
+console.log('value of isSparta:', isSparta);
+console.log('type of isSparta:', isSparta);
+
+var motto = fullName + ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:42]: Use the "Addition assignment operator" here (`+=`)

File: javascript-operators-and-expressions/main.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+var width = 420;
+var height = 70;
+var area = width * height;
+
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+var bill = 29400;
+var payment = 1073;
+var change = bill - payment;
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:10]: Shouldn't you pay enough to cover the bill? 😂 . It should be `payment - bill` and payment should be enough to cover the bill

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Lamberto!

File: javascript-operators-and-expressions/main.js (hunk 2 above)
Comment (Lambpato) [reply javascript-operators-and-expressions/main.js:10]: my bad im dyslexic

File: javascript-operators-and-expressions/main.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+console.log('value of is acidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+var headCount = 420;
+var isSparta = headCount === 300;
+
+console.log('value of isSparta:', isSparta);
+console.log('type of isSparta:', isSparta);
+
+var motto = fullName += ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:42]: By doing `fullName += ' is the GOAT';` you're reassigning this string to fullName, which is not want you want to do, you want this to be assigned to motto. So your code should look like this

`var motto = fullName;`
`motto += ' is the GOAT';`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change Lamberto!

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**